archivy/data.py

```diff
@@ -134,6 +134,15 @@ def get_item(dataobj_id):
     return None
 
 
+def move_item(dataobj_id, new_path):
+    """Move dataobj of given id to new_path"""
+    file = get_by_id(dataobj_id)
+    if move(file, f"{get_data_dir()}/{new_path}/"):
```

Comment:
we can just change this part to `return move(file, f"{get_data_dir()}/{new_path}/")`

Reply:
right.